src/testers.jl

```diff
@@ -144,21 +147,22 @@ end
 
 # Arguments
 - `config`: defaults to `ChainRulesTestUtils.ADviaRuleConfig`.
-- `f`: Function to which rule should be applied. Can also provide `f ⊢ f̄`.
-- `args` either the primal args `x`, or primals and their tangents: `x ⊢ x̄`
-    - `x`: input at which to evaluate `f` (should generally be set to an arbitary point in the domain).
-    - `x̄`: currently accumulated cotangent, will be generated automatically if not provided
+- `f`: function for which the `rrule` should be tested. Its tangent can be provided using `f ⊢ f̄`.
```

Comment:
Should this also be _co_tangent?
```suggestion
- `f`: function for which the `rrule` should be tested. Its cotangent can be provided using `f ⊢ f̄`.
```